catalog/core/catalog-core-backupplugin/src/main/java/ddf/catalog/backup/CatalogBackupPlugin.java

```diff
@@ -355,9 +358,42 @@ private String getExceptionMessage(String responseType, String previousException
         return builder.toString();
     }
 
+    public void shutdown() {
+        getPeriodicDrain().shutdown();
```

Comment:
Probably want to do something like this here:

```
 void shutdownAndAwaitTermination(ExecutorService pool) {
   pool.shutdown(); // Disable new tasks from being submitted
   try {
     // Wait a while for existing tasks to terminate
     if (!pool.awaitTermination(60, TimeUnit.SECONDS)) {
       pool.shutdownNow(); // Cancel currently executing tasks
       // Wait a while for tasks to respond to being cancelled
       if (!pool.awaitTermination(60, TimeUnit.SECONDS))
           System.err.println("Pool did not terminate");
     }
   } catch (InterruptedException ie) {
     // (Re-)Cancel if current thread also interrupted
     pool.shutdownNow();
     // Preserve interrupt status
     Thread.currentThread().interrupt();
   }
 }
```

See http://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ExecutorService.html for more details.